pkg/features/kube_features.go

```diff
@@ -743,6 +743,11 @@ const (
 	// pod's lifecycle and will not block pod termination.
 	SidecarContainers featuregate.Feature = "SidecarContainers"
 
+	// owner: @serathius
+	//
+	// Enables APF to use size of objects for estimating request cost.
+	SizeBasedListCostEstimate featuregate.Feature = "SizeBasedListCostEstimate"
```

Comment:
I don't think we want this - we should do the same thing we did with BtreeWatchCache (so not redefine it here, just rely on what's defined in apiserver already).